modules/@apostrophecms/ui/ui/apos/mixins/AposPublishMixin.js

```diff
@@ -49,9 +49,10 @@ export default {
         if ((e.name === 'invalid') && e.body && e.body.data && e.body.data.unpublishedAncestors) {
           if (await apos.confirm({
             heading: 'apostrophe:unpublishedParent',
-            description: 'apostrophe:unpublishedParentDescription',
+            description: 'apostrophe:unpublishedParentDescription'
+          }, {
             interpolate: {
-              unpublishedParents: this.$t(e.body.data.unpublishedAncestors.map(page => page.title).join(this.$t('apostrophe:listJoiner')))
+              unpublishedParents: e.body.data.unpublishedAncestors.map(page => page.title).join(this.$t('apostrophe:listJoiner'))
```

Comment:
This would always be a string of unknowable page titles, so there's no reason to pass through interpolation.

More importantly, the `interpolate` property needed to be in a options object (third argument), not the second `content` object.